gatsby-config.js

```diff
@@ -16,14 +16,13 @@ module.exports = {
     {
       resolve: '@newrelic/gatsby-theme-newrelic',
       options: {
-        gaTrackingId: 'UA-3047412-33',
```

Comment:
Are we positive we're not using this tracking ID for anything else?

Just want to make sure that the addition of Tessen (and Segment) replaces everything we were getting from Google Analytics.

Reply:
according to Zoe we no longer need this in the site config. I looked at the Docs site and how it was [implemented](https://github.com/newrelic/docs-website/wiki/Event-Data--and--Instrumentation)  and the way I understand it to work is: we rely on this configuration with tessen to pass all the data we need to segment and GA, and within Segment we have the GA tracking ID  configured. 

``` tessen: {
          product: 'DOC',
          subproduct: 'TDOC',
          segmentWriteKey: 'AEfP8c1VSuFxhMdk3jYFQrYQV9sHbUXx',
          trackPageViews: true,
          pageView: {
            name: 'pageView',
            category: 'DocPageView',
            getProperties: ({ location, env }) => ({
              path: location.pathname,
              env: env === 'production' ? 'prod' : env,
            }),
          },
        },

```